site/tests/app/entities/poll/PollTester.php

```diff
@@ -7,9 +7,8 @@
 use app\entities\poll\Response;
 use DateTime;
 use ReflectionProperty;
-use tests\BaseUnitTest;
 
-class PollTester extends BaseUnitTest {
+class PollTester extends \PHPUnit\Framework\TestCase {
```

Comment:
What necessitated this change?  It seems like we have a number of tests extending `BaseUnitTest` and a number of tests extending `TestCase`, so it's a plausible change.  I'm just wondering about the logic behind the decision.